antora.yml

```
@@ -63,6 +63,9 @@ asciidoc:
     newline: '<br>'
     virt_ellps: '&#8942;'
     ellps: '&#133;'
+    # Prevent Mergetags becoming variables
+    prefix: '{{'
+    suffix: '}}'
```

```suggestion
    open_double_curly_braces: '{{'
    closed_double_curly_braces: '}}'
```
Would these names reflect better its substitution?

---

I tried to keep them short, since they're going to be used in dozens of places. Prefix and Suffix are more accurate to their use within the demos, as they should only be used for merge tags.